JSDemos/Demos/TreeList/FocusedRow/description.md

```diff
@@ -1 +1,7 @@
-This example demonstrates a&nbsp;TreeList component that supports focused rows. The&nbsp;**focusedRowKey** property specifies the&nbsp;focused row. The **onFocusedRowChanged** callback function can be&nbsp;used to&nbsp;display additional information below the&nbsp;TreeList when the&nbsp;focused row changes. The **focusedRowEnabled** property enables row focusing.
\ No newline at end of file
+The TreeList component can highlight the focused row. To enable this feature, set the [focusedRowEnabled](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/#focusedRowEnabled) property to **true**.
+
+To focus a row programmatically, specify the row key in the [focusedRowKey](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/#focusedRowKey) property. The TreeList automatically scrolls to the focused row; you can set the [autoNavigateToFocusedRow](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/#autoNavigateToFocusedRow) property to **false** to deactivate this default behavior. In the UI, users can click a row to focus it. The focused row is saved in the TreeList's [state](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/stateStoring/).
```

Comment:
> in the [focusedRowKey](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/#focusedRowKey) property

Not recommended. The property name is self-explanatory. I think we can go with the DataGrid version.

```suggestion
To focus a row programmatically, specify the [focusedRowKey](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/#focusedRowKey) property. The TreeList automatically scrolls to the focused row. You can set the [autoNavigateToFocusedRow](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/#autoNavigateToFocusedRow) property to **false** to deactivate this behavior. In the UI, users can click a row to focus it. The focused row is saved in the TreeList's [state](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/stateStoring/).
```